conf/messages.cy

```diff
@@ -391,6 +391,13 @@ chargeSummary.whatYouOwe.textOne                                = Gweld
 chargeSummary.whatYouOwe.textTwo                                = i wirio a oes gennych unrhyw daliadau eraill sy'n ddyledus.
 chargeSummary.whatYouOwe.linkText                               = yr hyn sy'n ddyledus gennych
 chargeSummary.overdue                                           = YN HWYR
+chargeSummary.chargeHistory.heading								= Payment history
+chargeSummary.chargeHistory.date								= Date
+chargeSummary.chargeHistory.description							= Description
+chargeSummary.chargeHistory.amount								= Amount
+chargeSummary.chargeHistory.created.paymentOnAccount1.text		= Created payment on account 1 of 2
+chargeSummary.chargeHistory.created.paymentOnAccount2.text		= Created payment on account 2 of 2
+chargeSummary.chargeHistory.created.balancingCharge.text		= Created remaining balance
 
```

Comment:
Seem to have less messages in the cy file compared to the en file. Should align them (even if they are all in english for the moment)